README.md

```diff
@@ -66,14 +66,14 @@ Weekly Meeting Details:
 
 Since it currently a small group, we are some what flexible and move the meeting
 time around from time to time. Any meeting time/date changes will be broadcast
-on the Intel slack channel and updated meeting times will be posted in the
+on the NPWG slack channel and updated meeting times will be posted in the
```

Comment:
@martinkennelly Since you are here, could you change the Meeting info to the "K8s Network & Resource management projects weekly scrum updates" meeting (ie SR-IOV meeting). Standalone Userspace meeting was dropped since it was just Gary and myself and we talk about it in your meeting if needed.

Reply:
@Billy99 Done. Hope this is OK. LMK.